src/Controller/GameController.php

```diff
@@ -4,21 +4,63 @@
 
 use App\Entity\Board;
 use App\Services\GameService;
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Cookie;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 
 class GameController extends AbstractController
 {
-
+    const COOKIE_KEY = 'tipsy-game';
     private $gameService;
+    private $session;
 
-    public function __construct(GameService $gameService){
+    public function __construct(GameService $gameService, SessionInterface $session, LoggerInterface $logger)
+    {
         $this->gameService = $gameService;
+        $this->session = $session;
     }
+
     public function new()
     {
+        $game = $this->gameService->newGame();
+        $playerHash = $this->generatePlayerHash();
+        $response = $this->redirectToRoute('game');
+        $response->headers->setCookie(new Cookie($this::COOKIE_KEY, $playerHash));
+        $this->session->set($playerHash, $game);
+
+        return $response;
+    }
+
+    public function show(Request $request)
+    {
+        $playerHash = $request->cookies->get($this::COOKIE_KEY);
```

Comment:
What happens if there is no cookie?

Reply:
I now right? :P
Do you think I should redirect to `/` ? In order to create a new game... That's what seems best to me...